src/templates/podcast.js

```diff
@@ -1,16 +1,171 @@
+import ShareMenu from '~components/dev-hub/share-menu';
+
+import { addTrailingSlashIfMissing } from '~utils/add-trailing-slash-if-missing';
+import { useSiteMetadata } from '~hooks/use-site-metadata';
+
+import { lineHeight, screenSize, size } from '~components/dev-hub/theme';
+import { toDateString } from '~utils/format-dates';
+import { dateFormatOptions } from '~src/constants';
+
+const PODCAST_BREADCRUMB = [
+    {
+        label: 'Home',
+        target: '/',
+    },
+    { label: 'Learn', target: '/learn' },
+    {
+        label: 'Podcasts',
+        target: '/podcasts',
+    },
+];
+
+const TOOLTIP_TEXT = 'Podcast link copied to clipboard!';
+
+const Container = styled('div')`
+    margin: 0 auto;
+
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        justify-content: center;
+        margin-top: 100px;
+    }
+`;
+
+const Icons = styled('div')`
+    margin: 0 ${size.small};
+
+    span {
+        &:not(:first-of-type) {
+            margin-left: ${size.small};
+        }
+        padding: 0 ${size.tiny};
+    }
+
+    @media ${screenSize.largeAndUp} {
+        display: flex;
+        flex-direction: column;
+        margin: 0 ${size.default};
+
+        span:not(:first-of-type) {
+            margin-top: ${size.small};
+        }
+    }
+`;
+
+const Content = styled('article')`
+    margin: 0 auto;
+    max-width: ${size.maxContentWidth};
+    padding-left: ${size.small};
+    padding-right: ${size.small};
+    width: 100%;
+
+    @media ${screenSize.largeAndUp} {
+        margin: initial;
+    }
+`;
+
+const StyledParagraph = styled('p')`
+    line-height: ${lineHeight.default};
+`;
+
+const StyledPlayer = styled(AudioPlayer)`
+    margin-bottom: ${size.mediumLarge};
+
+    @media ${screenSize.largeAndUp} {
+        margin-bottom: 40px;
+    }
+`;
+
+const Podcast = ({
+    pageContext: {
+        data: {
+            url: podcastUrl,
+            description,
+            publishDate,
+            thumbnailUrl: image,
+            title,
+        },
+    },
+    path: slug,
+}) => {
+    const { siteUrl } = useSiteMetadata();
+    const pageUrl = addTrailingSlashIfMissing(`${siteUrl}${slug}`);
+    const formattedPublishDate = toDateString(publishDate, dateFormatOptions);
 
-const Podcast = ({ pageContext: { data: podcast, ...other } }) => {
-    console.log(other);
-    console.log(podcast);
     return (
         <Layout includeCanonical={false}>
-            Test Page
-            { podcast && <AudioPlayer podcast={podcast} /> }
+            <SEO
+                articleTitle={title}
+                canonicalUrl={pageUrl}
+                image={image}
+                metaDescription={description}
+                ogDescription={description}
+                ogTitle={title}
+                ogUrl={pageUrl}
+                twitter={{
+                    description,
+                    image,
+                    title,
+                }}
+                type="music.song"
```

Comment:
Let's stick with "article" here for now as "music.song" apparently can only be used by specific companies.